packages/volto/src/components/manage/Controlpanels/__snapshots__/Controlpanel.test.jsx.snap

```diff
@@ -9,12 +28,10 @@ exports[`Controlpanel renders a controlpanel component 1`] = `
       class="ui container"
     >
       <div
-        data-schema="{
-  \\"fieldsets\\": [],
-  \\"properties\\": {}
-}"
-        id="Form"
-      />
+        id="form"
+      >
+        requestError : [{'message': 'Twitter username should not include the "@" prefix character.', 'field': 'twitter_username', 'error': 'ValidationError'}]
```

Comment:
The string gets inside the generated HTML?

Reply:
No, this was a mock, just to know what value was passed to the `Form` component:

https://github.com/plone/volto/blob/0e627af4c6ce7afafd76b042aa0cb7cf82e9e20e/packages/volto/src/components/manage/Controlpanels/Controlpanel.test.jsx#L10-L15